shared/src/vyxal/elements/Elements.scala

```diff
@@ -1151,9 +1197,15 @@ object Elements:
       case VStr(str) => str + str.reverse
       case lst: VList => VList(lst ++ lst.reverse)
     },
-    addPart("Ͼ", Monad, false) {
-      case num: VNum => " " * num.toInt // hallelujah
-      case lst: VList => VList(lst.map(item => ListHelpers.sum(item.itr)))
+    "Ͼ" ->
+      direct(1) {
+        val top = pop()
+        top match
+          case num: VNum => push(" " * num.toInt) // hallelujah
+          case lst: VList => push(VList(lst.map(item => ListHelpers.sum(item.itr))))
+          case f: VFun if f.arity == 1 =>
+            RegisterHelpers.applyFn(f, RegisterHelpers.length - 1)
```

Comment:
We don't want to push the result of the function onto the stack?

Reply:
this is a port of `ᴿ` from v3 legacy

Reply:
Okay, I'll assume there was a good reason for that